src/odemis/util/transform.py

```diff
@@ -738,6 +739,15 @@ def __init__(
             numpy.zeros(2, dtype=float) if translation is None else translation
         )
 
+    def __matmul__(self, other):
```

Comment:
No typing info? other should be a GeometricTransform, and it'll return also a GeometricTransform, right?